### PR TITLE
Address comments.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: user_manual/Doxyfile [hunk 1]
```diff
@@ -832,7 +832,7 @@ EXCLUDE_SYMBOLS        =
 # that contain example code fragments that are included (see the \include
 # command).
 
-EXAMPLE_PATH           =
+EXAMPLE_PATH           = docs
```
Comment (nomis52) [inline user_manual/Doxyfile]: This is for the generator script that's coming soon.

File: doxygen/bootloader.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ protocol. This is a standard protocol for upgrading firmware over USB.
...
-Generally the update procedure is:
-- Download or create a new firmware image.
-- Activate the bootloader mode on the device.
-- Perform the update.
-- Run the new code.
+To update the bootloader you'll need special hardware and
+software that can interface with the PIC32 microcontroller.
+
+The easiest way to get started is with a PicKit 3 programmer. You'll also need
+to MPLAB IPE software.
```
Comment (peternewman) [inline doxygen/bootloader.md]: SPaG the

File: user_manual/docs/controller.md [hunk 3]
```diff
@@ -8,5 +8,14 @@ DMX / RDM Controller {#controller}
...
+
+For correct operation, RDM networks need to be
+[biased](https://en.wikipedia.org/wiki/Biasing) appropriately. This requires
+two things:
+
+- Biasing at the controller side.
+- A terminator on the last responder in the DMX chain.
+
+See your manufacturer's instructions for how to set the correct biasing for
+your device.
```
Comment (peternewman) [inline user_manual/docs/controller.md]: "when in controller mode."

Although I guess the switches also allow you to install a fixed terminator on the responder, but perhaps we shouldn't recommend that due to the confusion available?

File: user_manual/docs/rdm-responder.md [hunk 4]
```diff
@@ -92,7 +92,7 @@ Sensor Index | Name           | Type, Units                     | Units   | Note
 -------------|----------------|---------------------------------|---------|---------
 0            | Temperature    | Temperature, Centigrade         | Deci    | Data is from the onboard temp. sensor if present.
```
Comment (peternewman) [inline user_manual/docs/rdm-responder.md]: Should we add a note about this being random data if the sensor isn't fitted (although I still really think it should NAck in that case.

File: doxygen/bootloader.md (hunk 2 above)
Comment (nomis52) [inline doxygen/bootloader.md]: Done

File: user_manual/docs/controller.md (hunk 3 above)
Comment (nomis52) [inline user_manual/docs/controller.md]: Yeah, also this is the controller section.

File: user_manual/docs/rdm-responder.md (hunk 4 above)
Comment (nomis52) [inline user_manual/docs/rdm-responder.md]: Done

File: user_manual/docs/controller.md (hunk 3 above)
Comment (peternewman) [inline user_manual/docs/controller.md]: I meant to clarify when you're in controller mode.

File: user_manual/docs/main.md [hunk 5]
```diff
@@ -6,12 +6,12 @@ Introduction     {#mainpage}
...
 [Open Lighting Embedded](https://www.openlighting.org/ole/) (OLE) software. For
 developer information, including instructions on how to modify the OLE code,
-see the [OLE Developer Guide](http://docs.openlighting.org/ja-rule/doc/latest/).
+see the [OLE Developer Guide](http://docs.openlighting.org/ole/doc/latest/).
 
 Since the OLE software is customizable, the version running on a particular
 device may have different functionality from what is described here. Consult
-your manufacturer's documentation on the specific features enabled on a
-particular hardware product.
+your manufacturer's documentation on the features are enabled on
```
Comment (peternewman) [inline user_manual/docs/main.md]: SPaG "on which features are" or "on the features which are"

Comment (nomis52) [inline user_manual/docs/main.md]: Done in the next PR

**Pull request merged**